build-tools/libzip/libzip.targets

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_SourceTopDir>..\..</_SourceTopDir>
+  </PropertyGroup>
+  <UsingTask AssemblyFile="$(_SourceTopDir)\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" TaskName="Xamarin.Android.Tools.BootstrapTasks.GetNugetPackageBasePath" />
+  <Import Project="$(_SourceTopDir)\Configuration.props" />
+  <Import Project="libzip.props" />
+  <Import Project="libzip.projitems" />
+  <Target Name="_Make">
```

Comment:
To assist in that, you may want to consider a target like [`_SetAutogenShTimeToLastCommitTimestamp `](https://github.com/xamarin/xamarin-android/blob/0c073f6/build-tools/mono-runtimes/mono-runtimes.targets#L61-L66).